addon/helpers/number.js

```diff
@@ -3,6 +3,15 @@ import { MicroState } from 'ember-microstates';
 
 export default MicroState.extend({
 
+  initialValueFor([value = 0]) {
+    let num = Number(value);
+    if (`${num}` === 'NaN') {
```

Comment:
Phantom doesn't support `Number.isNaN`. 

> Is this what we want to do here? Getting too fancy with coercion is where many of the famous JavaScript WATs come from. I think we should just let NaN be NaN either that, or throw an exception.

The microstate will be broken if we allow `NaN` value. We should turn it into something that will continue to work but notify the user that a problem occurred. With current tooling, we'll have no way to provide user with context other than what value was passed into the helper. This is a much bigger discussion than we should have another time. 

What should I do here?